home/models/leaderboard.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+
+
+# Event model
+class Leaderboard(models.Model):
+    """ """
+
+    steps = models.IntegerField(help_text="Number of steps recorded")
+
+    account = models.ForeignKey(
+        "Account",
+        on_delete=models.CASCADE,
+        help_text="Account the data is linked to",
+    )
+    device = models.ForeignKey(
+        "Device",
+        on_delete=models.CASCADE,
+        help_text="Device the data is coming from",
+    )
+    contests = models.ForeignKey(
```

Comment:
Since this is now just a one-to-one association, this should be singular `contest`. The help_text should also be updated appropriately.